website/src/app/kb/deploy/sites/readme.mdx

```diff
@@ -1,17 +1,39 @@
+import PlanBadge from "@/components/PlanBadge";
+import SupportOptions from "@/components/SupportOptions";
+import NextStep from "@/components/NextStep";
+import Alert from "@/components/DocsAlert";
+
+<PlanBadge plans={["starter", "enterprise"]}>
+
 # Sites
 
-Sites represent a shared network environment that gateways and resources exist
-in. All gateways and resources in a site should be able to reach each other.
+</PlanBadge>
+
+Sites represent a shared network environment that Gateways and Resources exist
+within. All Gateways and Resources in a Site must have **unobstructed network
+connectivity** to each other.
```

Comment:
```suggestion
within. All Gateways in a Site must have **unobstructed network
connectivity** to all Resources in the Site.
```

The Gateways never connect to other Gateways and we don't need Resources to connect to each other since our scope is only that Gateways connect to Resources, right? Resources don't even need to be able to connect to Gateways if there's a trapdoor firewall for some reason.